benefits/enrollment_littlepay/session.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import time
+
+from django.http import HttpRequest
+
+logger = logging.getLogger(__name__)
+
+
+class Session:
+
+    _keys_access_token = "enrollment_littlepay_access_token"
+    _keys_access_token_expiry = "enrollment_littlepay_access_token_expiry"
```

Comment:
Nit: we can probably leave `enrollment` out of the keys here. But this is a very small deal.